.travisci/coverage.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+mvn jacoco:report coveralls:report -DrepoToken=${repoToken}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/powerunit/powerunit-extensions-matchers/pullRequest?prid=5574175)